eng/pipelines/daily-verification.yml

```diff
@@ -3,8 +3,8 @@ jobs:
   - job:
     displayName: Daily vcpkg PR
     pool:
-      name: azsdk-pool-mms-win-2019-general
-      vmImage: MMS2019
+      name: azsdk-pool-mms-win-2022-general
+      vmImage: MMS2022
```

Comment:
`vmImage` can be removed here. It's not used directly.

Reply:
I'll clean this up in another issue. https://github.com/Azure/azure-sdk-for-cpp/issues/4211